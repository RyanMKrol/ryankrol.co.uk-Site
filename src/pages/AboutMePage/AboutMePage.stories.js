import React from 'react';

import AboutMePage from './AboutMePage';

export default {
  title: 'Pages/AboutMePage',
};

export const Default = () => <AboutMePage />;
