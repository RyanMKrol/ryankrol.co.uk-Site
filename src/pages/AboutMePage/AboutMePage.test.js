import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import AboutMePage from './AboutMePage';

it('renders AboutMePage correctly', () => {
  const header = shallow(<AboutMePage />);
  expect(toJson(header)).toMatchSnapshot();
});
