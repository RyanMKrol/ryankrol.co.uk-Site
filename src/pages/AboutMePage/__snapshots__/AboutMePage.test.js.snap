// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AboutMePage correctly 1`] = `
<div>
  <p
    className="highlighted"
  >
     Hi! My name is 
  </p>
  <h1
    className="light name"
  >
    Ryan Krol.
  </h1>
  <h1
    className="highlighted"
  >
    Welcome to my website!
  </h1>
  <p>
    I'm a London-based software engineer, currently working at Amazon UK building and maintaining acquisition experiences for PrimeVideo. If you have ever paid for a film/tv show that was streamed on Amazon, the chances are you've used my code!
  </p>
  <p>
    In my spare time, I like to hack together side projects just like this website, and many others that will no doubt be detailed somewhere on this site.
  </p>
  <p>
    I've recently been enjoying keeping track of the films and albums I've been enjoying, so if you care about my opinion on anything, you can find what I think of those here too :D
  </p>
</div>
`;
